# Quantitative modeling of multi-signal quorum sensing maps environment to bacterial regulatory responses

## Supporting Information

### Literature Search

| Paper | PMID | â†’*lasI* | â†’*lasR* | â†’*rhlI* | â†’*rhlR* | â†’elastase |
|----|:--:|:--:|:--:|:--:|:--:|:--:|
| GarcÃ­a-Reyes, SoberÃ³n-ChÃ¡vez, and Cocotl-Yanez 2020\[1\] | [31794380](https://pubmed.ncbi.nlm.nih.gov/31794380/) | â— | â—‹ | â— | â— | â— |
| Rutherford and Bassler 2012\[2\] | [23125205](https://pubmed.ncbi.nlm.nih.gov/23125205/) | â— | â—‹ | â— | â— |  |
| Proctor, McCarron, and Ternan 2020\[3\] | [31971503](https://pubmed.ncbi.nlm.nih.gov/31971503/) | â— | â—‹ | â—‹ | â— |  |
| Jakobsen et al.Â 2013\[4\] | [23841636](https://pubmed.ncbi.nlm.nih.gov/23841636/) | â— | â—‹ | â— | â— |  |
| Soukarieh et al.Â 2018\[5\] | [29999316](https://pubmed.ncbi.nlm.nih.gov/29999316/) | â— | â— | â— | â— |  |
| Tateda 2005\[6\] | [15926474](https://pubmed.ncbi.nlm.nih.gov/15926474/) | â— | â—‹ | â— | â—‹ | â— |
| Williams et al.Â 2007\[7\] | [19249239](https://pubmed.ncbi.nlm.nih.gov/19249239/) | â—‹ | â—‹ | â— | â— |  |
| Heurlier, DÃ©nervaud, and Haas 2006\[8\] | [16503417](https://pubmed.ncbi.nlm.nih.gov/16503417/) | â— | â—‹ | â— | â—‹ |  |
| Le Berre et al.Â 2006\[9\] | [16631332](https://pubmed.ncbi.nlm.nih.gov/16631332/) | â—‹ | â—‹ | â— | â— |  |
| Juhas, Eberl, and TÃ¼mmler 2005\[10\] | [15816912](https://pubmed.ncbi.nlm.nih.gov/15816912/) | â— | â— | â— | â— | â— |
| Donabedian 2003\[11\] | [12799145](https://pubmed.ncbi.nlm.nih.gov/12799145/) | â— | â—‹ | â— | â— | â— |
| Reuter, Steinbach, and Helms 2016\[12\] | [26819549](https://pubmed.ncbi.nlm.nih.gov/26819549/) | â— | â—‹ | â—‹ | â— | â— |
| Yong and Zhong 2013\[13\] | [22767136](https://pubmed.ncbi.nlm.nih.gov/22767136/) | â— | â—‹ | â— | â— | â— |
| Welsh and Blackwell 2016\[14\] | [27268906](https://pubmed.ncbi.nlm.nih.gov/27268906/) |  |  | â— | â— | â— |
| De Sordi and MÃ¼hlschlegel 2009\[15\] | [19845041](https://pubmed.ncbi.nlm.nih.gov/19845041/) | â— | â—‹ | â— | â—‹ |  |
| Winzer and Williams 2001\[16\] | [11437336](https://pubmed.ncbi.nlm.nih.gov/11437336/) | â— | â—‹ | â—‹ | â— | â— |
| Schuster et al.Â 2013\[17\] | [23682605](https://pubmed.ncbi.nlm.nih.gov/23682605/) | â— | â—‹ | â— | â— |  |
| Papaioannou, Utari, and Quax 2013\[18\] | [24065108](https://pubmed.ncbi.nlm.nih.gov/24065108/) | â— | â— | â—‹ | â— | â— |
| Roy, Adams, and Bentley 2011\[19\] | [22112397](https://pubmed.ncbi.nlm.nih.gov/22112397/) | â— | â—‹ | â— | â— |  |

**Table A. Activation of QS genes by LasR/3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL in review of published literature.** Solid dots indicate positive activation in the paperâ€™s diagram of gene transcription, while hollow dots indicate that the diagram shows no effect. No diagrams indicated repression. Note that some papers made no attempt to indicate particular interactions; several, for example, concentrated strictly on the QS genes themselves and did not show the effect on downstream genes such as those for elastase.

| Paper | PMID | â†’*lasI* | â†’*lasR* | â†’*rhlI* | â†’*rhlR* | â†’elastase |
|----|:--:|:--:|:--:|:--:|:--:|:--:|
| GarcÃ­a-Reyes, SoberÃ³n-ChÃ¡vez, and Cocotl-Yanez 2020\[1\] | [31794380](https://pubmed.ncbi.nlm.nih.gov/31794380/) | â—‹ | â—‹ | â—‹ | â—‹ | â— |
| Rutherford and Bassler 2012\[2\] | [23125205](https://pubmed.ncbi.nlm.nih.gov/23125205/) | â—‹ | â—‹ | â— | â—‹ |  |
| Proctor, McCarron, and Ternan 2020\[3\] | [31971503](https://pubmed.ncbi.nlm.nih.gov/31971503/) | â—‹ | â—‹ | â— | â—‹ |  |
| Jakobsen et al.Â 2013\[4\] | [23841636](https://pubmed.ncbi.nlm.nih.gov/23841636/) | â—‹ | â—‹ | â— | â—‹ |  |
| Soukarieh et al.Â 2018\[5\] | [29999316](https://pubmed.ncbi.nlm.nih.gov/29999316/) | â—‹ | â—‹ | â—‹ | â—‹ |  |
| Tateda 2005\[6\] | [15926474](https://pubmed.ncbi.nlm.nih.gov/15926474/) | â—‹ | â—‹ | â— | â—‹ | â— |
| Williams et al.Â 2007\[7\] | [19249239](https://pubmed.ncbi.nlm.nih.gov/19249239/) | â—‹ | â—‹ | â—‹ | â—‹ |  |
| Heurlier, DÃ©nervaud, and Haas 2006\[8\] | [16503417](https://pubmed.ncbi.nlm.nih.gov/16503417/) | â—‹ | â—‹ | â— | â—‹ |  |
| Le Berre et al.Â 2006\[9\] | [16631332](https://pubmed.ncbi.nlm.nih.gov/16631332/) | â—‹ | â—‹ | â—‹ | â—‹ |  |
| Juhas, Eberl, and TÃ¼mmler 2005\[10\] | [15816912](https://pubmed.ncbi.nlm.nih.gov/15816912/) | â—‹ | â—‹ | â— | â— | â— |
| Donabedian 2003\[11\] | [12799145](https://pubmed.ncbi.nlm.nih.gov/12799145/) | â—‹ | â—‹ | â—‹ | â—‹ | â— |
| Reuter, Steinbach, and Helms 2016\[12\] | [26819549](https://pubmed.ncbi.nlm.nih.gov/26819549/) | â—‹ | â—‹ | â—‹ | â—‹ | â— |
| Yong and Zhong 2013\[13\] | [22767136](https://pubmed.ncbi.nlm.nih.gov/22767136/) | â—‹ | â—‹ | â— | â—‹ | â— |
| Welsh and Blackwell 2016\[14\] | [27268906](https://pubmed.ncbi.nlm.nih.gov/27268906/) | â—‹ | â—‹ |  |  | â—‹ |
| De Sordi and MÃ¼hlschlegel 2009\[15\] | [19845041](https://pubmed.ncbi.nlm.nih.gov/19845041/) | â—‹ | â—‹ | â— | â—‹ |  |
| Winzer and Williams 2001\[16\] | [11437336](https://pubmed.ncbi.nlm.nih.gov/11437336/) | â—‹ | â—‹ | â— | â—‹ | â— |
| Schuster et al.Â 2013\[17\] | [23682605](https://pubmed.ncbi.nlm.nih.gov/23682605/) | â—‹ | â—‹ | â— | â—‹ |  |
| Papaioannou, Utari, and Quax 2013\[18\] | [24065108](https://pubmed.ncbi.nlm.nih.gov/24065108/) | â—‹ | â—‹ | â— | â—‹ | â— |
| Roy, Adams, and Bentley 2011\[19\] | [22112397](https://pubmed.ncbi.nlm.nih.gov/22112397/) | â—‹ | â—‹ | â—‹ | â—‹ |  |

**Table B. Activation of QS genes by RhlR/C<sub>4</sub>â€‘HSL in review of published literature.** Same notation as previous table.

### Data Analysis

Gene expression data for *lasI,* *rhlI,* and *lasB* was collected every hour for a 24-hour period. Observations used for analysis were limited to a two-hour window that contained the peak expression level for each gene. Figs A, B, and C show the full time course of expression levels and highlight the intervals used for analysis. Those windows were 8â€“10 hours, 3â€“5 hours, and 4â€“6 hours for *lasI,* *rhlI,* and *lasB,* respectively.

<img src="Figures/figs1_lasi_time.svg" data-number="1" alt="Figure A: lasi_time" />


**Figure A. Expression level of *lasI* over time course of experiment.** Shaded regions highlight peak expression and indicate two-hour period used in analysis. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

<img src="Figures/figs2_rhli_time.svg" alt="Figure B: rhli_time" />


**Figure B. Expression level of *rhlI* over time course of experiment.** Shaded regions highlight peak expression and indicate two-hour period used in analysis. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

<img src="Figures/figs3_lasb_time.svg" alt="Figure C: lasb_time" />


**Figure C. Expression level of *lasB* over time course of experiment.** Shaded regions highlight peak expression and indicate two-hour period used in analysis. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

### Single-Signal Models

Table C shows the parameter estimates for the single-signal model below as maximum fold-change ((*É‘* + *É‘*<sub>0</sub>) / *É‘*<sub>0</sub>) and half-concentration values (*K*) for both signals.
``` math
E(S) = \alpha_0 + \alpha \frac{[S]}{[S] + K}
\qquad{(\mathrm{A})}
```

**Equation A. Expression level as a function of a single signalâ€™s concentration.**

| Gene | Signal | Parameter | Derivation | Estimate | 95% C.I. |
|----|----|----|:--:|:--:|---:|
| *lasI* |  | Basal expression | *É‘*<sub>0</sub> | 1670 <small>RLU/OD</small> | 1619 â€“ 1721 |
|  | 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘* + *É‘*<sub>0</sub>) / *É‘*<sub>0</sub> | 38 Ã— | 36 â€“ 40 |
|  |  | Â½ conc. | *K* | 0.24 Î¼M | 0.17 â€“ 0.30 |
|  | C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘* + *É‘*<sub>0</sub>) / *É‘*<sub>0</sub> | 6.4 Ã— | 5.8 â€“ 7.0 |
|  |  | Â½ conc. | *K* | 1.0 Î¼M | 0.7 â€“ 1.4 |
| *rhlI* |  | Basal expression | *É‘<sub>0</sub>* | 1861 <small>RLU/OD</small> | 1798 â€“ 1923 |
|  | 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘* + *É‘*<sub>0</sub>) / *É‘*<sub>0</sub> | 35 Ã— | 34 â€“ 36 |
|  |  | Â½ conc. | *K* | 0.052 Î¼M | 0.031 â€“ 0.073 |
|  | C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘* + *É‘*<sub>0</sub>) / *É‘*<sub>0</sub> | 6.4 Ã— | 5.3 â€“ 7.4 |
|  |  | Â½ conc. | *K* | 1.6 Î¼M | 0.8 â€“ 2.4 |

**Table C. Single Signal Parameter Estimates.** Estimated fold-change, derived from raw parameters of Eq A as (*É‘* + *É‘*<sub>0</sub>) / *É‘*<sub>0</sub> , and half-concentration, *K*, values for gene expression as a function of a single signal in isolation. Values shown with 95% confidence intervals.

The primary data set focuses on a full range of signal concentrations from 0 to 5Î¼M. To further validate the model, additional measurements were collected for low values of signal concentration. Fig D overlays those observations on the primary data set, demonstrating further strong agreement between observations and model predictions.

<img src="Figures/figs4_lowconc.svg" alt="Figure D: model1" />


**Figure D. Effect of each signal in isolation on the expression level of *lasI*** Plotted points are observations and dashed lines show model (Eq A) predictions when parameterized per Table C. Dark blue points are additional observations collected using low signal concentrations. (A single data point identified as a faulty outlier is indicated in light blue and excluded from the analysis.) These data points are not used in estimating model parameters yet still show strong agreement with the model. Coefficient of determination R<sup>2</sup> between additional observations and initial model predictions is 0.82. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

### Multi-Signal Models

Table D shows the parameter estimates for the multi-signal model of Eq 1 (main text).

| Gene | Signal | Parameter | Derivation | Estimate | 95% C.I. |
|----|----|----|:--:|:--:|---:|
| *lasI* |  | Basal expression | *É‘*<sub>1,0</sub> | 1670 <small>RLU/OD</small> | 1619 â€“ 1721 |
|  | 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘*<sub>1,1</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 38 Ã— | 36 â€“ 40 |
|  |  | Â½ conc. | *K*<sub>1,1</sub> | 0.24 Î¼M | 0.17 â€“ 0.30 |
|  | C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘*<sub>1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 6.4 Ã— | 5.8 â€“ 7.0 |
|  |  | Â½ conc. | *K*<sub>1,2</sub> | 1.0 Î¼M | 0.7 â€“ 1.4 |
|  | Combined | Max fold-change | (*É‘*<sub>1,1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 30 Ã— | 29 â€“ 31 |
|  |  | Â½ conc. for 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | *K*<sub>*Q*1,1,2</sub> | \< 0.001 Î¼M |  |
|  |  | Â½ conc. for C<sub>4</sub>-HSL | *K*<sub>*Q*1,2,1</sub> | 0.003 Î¼M | 0 â€“ 0.011 |
| *rhlI* |  | Basal expression | *É‘*<sub>2,0</sub> | 1861 <small>RLU/OD</small> | 1798 â€“ 1923 |
|  | 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘*<sub>2,1</sub> + *É‘*<sub>2,0</sub>) / *É‘*<sub>2,0</sub> | 35 Ã— | 34 â€“ 36 |
|  |  | Â½ conc. | *K*<sub>2,1</sub> | 0.052 Î¼M | 0.031 â€“ 0.073 |
|  | C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘*<sub>2,2</sub> + *É‘*<sub>2,0</sub>) / *É‘*<sub>2,0</sub> | 6.4 Ã— | 5.3 â€“ 7.4 |
|  |  | Â½ conc. | *K*<sub>2,2</sub> | 1.6 Î¼M | 0.8 â€“ 2.4 |
|  | Combined | Max fold-change | (*É‘*<sub>2,1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 27 Ã— | 26 â€“ 28 |
|  |  | Â½ conc. for 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | *K*<sub>*Q*2,1,2</sub> | \< 0.001 Î¼M |  |
|  |  | Â½ conc. for C<sub>4</sub>-HSL | *K*<sub>*Q*2,2,1</sub> | \< 0.001 Î¼M |  |

**Table D. Multi-signal parameter estimates.** Model parameters for gene expression as a function of multiple signal concentrations. Parameter definitions are the same as in Table C with addition of cooperative fold-change, again derived from raw parameters as (*É‘* + *É‘*<sub>0</sub>) / *É‘*<sub>0</sub> , and cooperative half-concentration *K<sub>Q</sub>.* Values shown with 95% confidence intervals.

Figs 3C and 3D in the main text summarize the predictions of the multi-signal models for *lasI* and *rhlI* expression. The following figures provide a more detailed comparison of the model predictions for both genes.

<img src="Figures/figs5_model_lasi.svg" alt="Figure E: model_lasi" />


**Figure E. Multi-signal model for *lasI* expression.** Panels compare model predictions to observations for all combinations of signal concentrations. Horizontal bars indicate model predictions, while plotted points show observed values. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

<img src="Figures/figs6_model_rhli.svg" alt="Figure F: model_rhli" />


**Figure F. Multi-signal model for *rhlI* expression.** Panels compare model predictions to observations for all combinations of signal concentrations. Horizontal bars indicate model predictions, while plotted points show observed values. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

Table E shows the parameter estimates for *lasB* expression.

| Signal | Parameter | Derivation | Estimate | 95% C.I. |
|----|----|:--:|:--:|---:|
|  | Basal Expression | *É‘*<sub>3,0</sub> | 1588 <small>RLU/OD</small> | 1516 â€“1660 |
| 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘*<sub>3,1</sub> + *É‘*<sub>3,0</sub>) / *É‘*<sub>3,0</sub> | 6.1 Ã— | 5.6 â€“ 6.7 |
|  | Â½ conc. | *K*<sub>3,1</sub> | 2.5 Î¼M | 1.0 â€“ 3.0 |
| C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘*<sub>3,2</sub> + *É‘*<sub>3,0</sub>) / *É‘*<sub>3,0</sub> | 1.1 Ã— | 1.1 â€“ 1.1 |
|  | Â½ conc. | *K*<sub>3,2</sub> | \< 0.001 Î¼M |  |
| Combined | Max fold-change | (*É‘*<sub>3,1,2</sub> + *É‘*<sub>3,0</sub>) / *É‘*<sub>3,0</sub> | 23 Ã— | 22 â€“ 24 |
|  | Â½ conc. for 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | *K*<sub>*Q*3,1,2</sub> | 0.42 Î¼M | 0.35 â€“ 0.48 |
|  | Â½ conc. for C<sub>4</sub>-HSL | *K*<sub>*Q*3,2,1</sub> | 0.22 Î¼M | 0.18 â€“ 0.25 |

**Table E. Multi-signal parameter estimates for *lasB.*** Model parameters for *lasB* expression as a function of multiple signal concentrations. Parameter definitions are the same as in Table D. Values shown with 95% confidence intervals. Half-concentration estimates less than 0.001 Î¼M are below the limits of precision of the experimental data.

Using the parameter values, the model predicts *lasB* expression as shown in Fig G.

<img src="Figures/figs7_model_lasb.svg" alt="Figure G: model_lasb" />


**Figure G. Multi-signal model for *lasB* expression.** Panels compare model predictions to observations for all combinations of signal concentrations. Horizontal bars indicate model predictions, while plotted points show observed values. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

### Signal Dynamics

We analyze signal dynamics using the model from the main text where the per-capita single production rater is assumed to be proportional to the synthase expression level, $`E_i(\mathbf{S})`$. The proportionality constant is $`c_i`$.

``` math
\frac{\mathrm{d}S_i}{\mathrm{dt}} \ \ = \ \ c_i E_i(\mathbf{S})\cdot N \ \ - \ \  \delta_i \cdot S_i \ \ - \ \  m \cdot S_i
\qquad{(\mathrm{B})}
```

We consider the equilibrium signal concentration (where $`\mathrm{d}S/\mathrm{dt} = 0`$) and normalize to the decay rate of C<sub>4</sub>â€‘HSL ($`\delta_2`$). When there is no mass transfer ($`m = 0`$), these simplifications result in an equation for C<sub>4</sub>â€‘HSL,

``` math
0 \ \ = \ \ \frac{ c_2 }{ \delta_2 } E_2(\mathbf{S})\cdot N \ \ - \ \ S_2
\qquad{(\mathrm{C})}
```

which can be solved for $`c_2 / \delta_2`$ in terms of *rhlI* expression $`E_2(\mathbf{S})`$, density $`N`$, and C<sub>4</sub>â€‘HSL concentration $`S_2`$. The corresponding equation for 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL includes an additional factor $`\delta_1/\delta_2`$ which, from \[20\], we take to be approximately 1.7.

``` math
0 \ \ = \ \ \frac{ c_1 }{ \delta_2 } E_1(\mathbf{S})\cdot N \ \ - \ \ \frac{ \delta_1 }{ \delta_2 } S_1
\qquad{(\mathrm{D})}
```

Data from \[21\] includes measurements of equilibrium signal concentrations at multiple population densities. We combine those measurements of $`N`$ and $`S_i`$ with our modelâ€™s estimate of synthase expression level $`E_i(\mathbf{S})`$ and use non-linear least squares to estimate the proportionality constants.

| Signal *i*                   | Proportionality Constant $`c_i/\delta_2`$ |
|:-----------------------------|------------------------------------------:|
| **3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL** |                               12.7 pM/RLU |
| **C<sub>4</sub>â€‘HSL**        |                               25.4 pM/RLU |

**Table F. Estimated proportionality constants that relate synthase expression levels to per-capita signal production rates.** Final column shows adjusted R<sup>2</sup> of non-linear least squares estimate.

<img src="Figures/figs8_constants.svg" alt="Figure H: constants" />


**Figure H. Equilibrium signal concentration predicted using proportionality constants.** Individual data points show experimental observations and dashed lines indicate model predictions. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

### Analytic Solutions for Equilibrium

It is possible to derive analytic solutions of Eq 2 (main text) for equilibrium concentrations in all architectures; however, the results are not especially helpful for deriving insights into the system behavior. For example, the independent architecture, which is the simplest considered, has the following equilibrium concentration of 3â€‘oxoâ€‘C <sub>12</sub>â€‘HSL.
``` math
S_1^* = \frac{
\begin{align}
c_1\,N\,(\alpha_{1,0} +\alpha_{1,1}) \\ - K_{1,1}\,(m+\delta_1)
\end{align}
\;+\sqrt{
\begin{aligned}
{K_{1,1} }^2 \,{\delta_1 }^2 +2\,{K_{1,1} }^2 \,\delta_1 \,m+{K_{1,1} }^2 \,m^2
+2\,K_{1,1} \,N\,\alpha_{1,0} \,c_1 \,\delta_1 \\
+2\,K_{1,1} \,N\,\alpha_{1,0} \,c_1 \,m -2\,K_{1,1} \,N\,\alpha_{1,1} \,c_1 \,\delta_1 -2\,K_{1,1} \,N\,\alpha_{1,1} \,c_1 \,m \\
+N^2 \,{\alpha_{1,0} }^2 \,{c_1 }^2 +2\,N^2 \,\alpha_{1,0} \,\alpha_{1,1} \,{c_1 }^2
+N^2 \,{\alpha_{1,1} }^2 \,{c_1 }^2
\end{aligned}
}
}{2\,{\left(\delta_1 +m\right)}}
\qquad{(\mathrm{E})}
```

### Alternate QS Architectures

Table G shows the parameter values that allow Eq 1 (main text) to represent various QS architectures.

| Gene | Signal | Parameter | Derivation | Reciprocal Architecture | Hierarchical Architecture | Independent Architecture |
|----|----|----|:--:|:--:|:--:|:--:|
| *lasI* | 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘*<sub>1,1</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 38 Ã— | 38 Ã— | 38 Ã— |
|  | C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘*<sub>1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 6.4 Ã— | 1 Ã— | 1 Ã— |
|  | Combined | Max fold-change | (*É‘*<sub>1,1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 30 Ã— | 1 Ã— | 1 Ã— |
| *rhlI* | 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘*<sub>2,1</sub> + *É‘*<sub>2,0</sub>) / *É‘*<sub>2,0</sub> | 35 Ã— | 35 Ã— | 1 Ã— |
|  | C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘*<sub>2,2</sub> + *É‘*<sub>2,0</sub>) / *É‘*<sub>2,0</sub> | 6.4 Ã— | 6.4 Ã— | 6.4 Ã— |
|  | Combined | Max fold-change | (*É‘*<sub>2,1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 27 Ã— | 27 Ã— | 1 Ã— |

**Table G. Hierarchical and independent architectures are special cases of the reciprocal architecture.** The multi-signal model of Eq 1 (main text) can represent hypothetical, alternative QS architectures by setting appropriate *É‘* values to zero. Zero *É‘* values result in a corresponding maximum fold-change of 1. For a hierarchical architecture, this setting nullifies the effect of C<sub>4</sub>â€‘HSL on *lasI.* For an independent architecture, this setting additionally nullifies the effect of 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL on *rhlI.*

### Normalizing Alternate QS Architectures

Table G analyzes hypothetical, alternative architectures by eliminating the influence of specific signals on specific genes. For example, the hierarchical architecture nullifies the influence of C<sub>4</sub>â€‘HSL on *lasI* without modifying the effect of 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL on *lasI.* This change necessarily reduces the maximum expression level of *lasI,* and that reduction partially explains the different *lasB* response in a hierarchical architecture. Reducing maximum *lasI* expression alone, however, does not explain all of the differences in the *lasB* response. To expose those additional differences, we make additional adjustments to the model. In particular, we increase the expression of *lasI* due to 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL to precisely compensate for the loss of expression due to C<sub>4</sub>â€‘HSL. Table H shows the full set of adjustments required to normalize the maximum synthase expression levels across all architectures.

| Gene | Signal | Parameter | Derivation | Reciprocal Architecture | Hierarchical Architecture | Independent Architecture |
|----|----|----|:--:|:--:|:--:|:--:|
| *lasI* | 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘*<sub>1,1</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 38 Ã— | 73 Ã— | 73 Ã— |
|  | C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘*<sub>1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 6.4 Ã— | 1 Ã— | 1 Ã— |
|  | Combined | Max fold-change | (*É‘*<sub>1,1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 30 Ã— | 1 Ã— | 1 Ã— |
| *rhlI* | 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL | Max fold-change | (*É‘*<sub>2,1</sub> + *É‘*<sub>2,0</sub>) / *É‘*<sub>2,0</sub> | 35 Ã— | 35 Ã— | 1 Ã— |
|  | C<sub>4</sub>â€‘HSL | Max fold-change | (*É‘*<sub>2,2</sub> + *É‘*<sub>2,0</sub>) / *É‘*<sub>2,0</sub> | 6.4 Ã— | 6.4 Ã— | 66 Ã— |
|  | Combined | Max fold-change | (*É‘*<sub>2,1,2</sub> + *É‘*<sub>1,0</sub>) / *É‘*<sub>1,0</sub> | 27 Ã— | 27 Ã— | 1 Ã— |

**Table H. Models of hierarchical and independent architectures can be normalized to ensure that maximum synthase expression is the same for all architectures.** Parameters are the same as those in Table G but with increased values where appropriate.

### Temporal Dynamics

<img src="Figures/figs9_lasb_time_response.svg" alt="Figure I: lasb_time_response" />


**Figure I. Time response of *lasB* expression for reciprocal, rescaled hierarchical, and rescaled independent architectures.** Dynamics are those of Eq 2 (main text) with parameters from Table H. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

### Signal Concentration Response

<img src="Figures/figs10_2signals.svg" alt="Figure J: 2signals" />


**Figure J. Extracellular signal concentration as a function of density and mass transfer varies based on the quorum sensing architecture.** Heat maps of equilibrium 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL (A-C) and C<sub>4</sub>â€‘HSL (D-F) concentration for three quorum sensing architectures. Both population density and mass transfer rate are varied over the same ranges for all heatmaps. The lines on each heat map indicate density and mass transfer values for which equilibrium concentration is constant, either 50% of its maximum value (white) or 5% of its maximum value (black). Equilibrium concentrations calculated from Eq 2 model with parameters from Table D and architectural parameters normalized according to Table H. These results follow from the same model parameterization presented in Figs 5F-5H, which showcased the predicted outcome behavior of *lasB* expression. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

<img src="Figures/figs11_signal_ratio.svg" alt="Figure K: signal_ratio" />


**Figure K. Ratio of signal concentrations as a function of density and mass transfer varies based on the quorum sensing architecture.** The figure shows heat maps of the ratio of equilibrium 3â€‘oxoâ€‘C<sub>12</sub>â€‘HSL to C<sub>4</sub>â€‘HSL concentration for the reciprocal architecture. Equilibrium concentrations calculated from Eq 2 model with parameters from Table D. (The data underlying this Figure and the code used to analyze it can be found in https://doi.org/10.5281/zenodo.15808353.)

### Three Signal Models

The hypothetical three-signal models of the main textâ€™s discussion are based oh a simplified version of the *las* and *rhl* interactions. Table D provides the starting point for the models. For ease of computation, the second-order effects are ignored by setting *É‘<sub>i,j,j\`</sub>* to zero. Parameters for the third signal (*i* = 3) are initailly based on convenient intermediate values between those of *las* and *rhl* and then varied as neccessary to demontrate the various responses. Table I shows the values for all non-zero parameters in all models.

| Parameter | Weak | Strong | Limited | Damped |
|----|----|----|----|----|
| ğ›¼<sub>1,0</sub> | 1670 | 1670 | 1670 | 1670 |
| ğ›¼<sub>2,0</sub> | 1861 | 1861 | 1861 | 1861 |
| ğ›¼<sub>3,0</sub> | 10000 | 10000 | 10000 | 10000 |
| ğ›¼<sub>1,1</sub> | 61000 | 61000 | 61000 | 61000 |
| ğ›¼<sub>2,2</sub> | 10000 | 10000 | 10000 | 10000 |
| ğ›¼<sub>3,3</sub> | 10000 | 10000 | 10000 | 10000 |
| ğ›¼<sub>1,2</sub> | 9000 | 9000 | 9000 | 9000 |
| ğ›¼<sub>2,1</sub> | 63000 | 63000 | 63000 | 63000 |
| ğ›¼<sub>2,3</sub> | 10000 | 10000 | 10000 | 10000 |
| ğ›¼<sub>3,1</sub> | 10000 | 1000000 | 158000 | 630000 |
| ğ›¼<sub>3,2</sub> | -10000 | -10000 | -245 | -1580000 |
| K<sub>1,1</sub> | 0.24 | 0.24 | 0.24 | 0.24 |
| K<sub>2,2</sub> | 1.6 | 1.6 | 1.6 | 1.6 |
| K<sub>3,3</sub> | 1 | 1 | 1 | 1 |
| K<sub>1,2</sub> | 1 | 1 | 1 | 1 |
| K<sub>2,1</sub> | 0.052 | 0.052 | 0.052 | 0.052 |
| K<sub>2,3</sub> | 0.32 | 0.32 | 0.32 | 0.32 |
| K<sub>3,1</sub> | 0.32 | 0.32 | 0.032 | 0.032 |
| K<sub>3,2</sub> | 0.32 | 0.32 | 4.0 | 3.2 |
| c<sub>1</sub>/ğ›¿<sub>2</sub> | 1.3â¨‰10<sup>-5</sup> | 1.3â¨‰10<sup>-5</sup> | 1.3â¨‰10<sup>-5</sup> | 1.3â¨‰10<sup>-5</sup> |
| c<sub>2</sub>/ğ›¿<sub>2</sub> | 2.5â¨‰10<sup>-5</sup> | 2.5â¨‰10<sup>-5</sup> | 2.5â¨‰10<sup>-5</sup> | 2.5â¨‰10<sup>-5</sup> |
| c<sub>3</sub>/ğ›¿<sub>2</sub> | 1.9â¨‰10<sup>-5</sup> | 1.9â¨‰10<sup>-5</sup> | 1.9â¨‰10<sup>-5</sup> | 1.9â¨‰10<sup>-5</sup> |
| ğ›¿<sub>1</sub>/ğ›¿<sub>2</sub> | 1.7 | 1.7 | 1.7 | 1.7 |
| ğ›¿<sub>2</sub>/ğ›¿<sub>2</sub> | 1 | 1 | 1 | 1 |
| ğ›¿<sub>3</sub>/ğ›¿<sub>2</sub> | 1.35 | 1.35 | 1.35 | 1.35 |

**Table I. Model parameters for hypothetical three-signal architectures.** Different parameter values result in the different responses of the third QS systemâ€™s synthase expression level as population density increases.

<img src="Figures/figs12_3signals.svg" alt="Figure L: 3signals" />


**Figure L. Interaction strength for both induction and repression determines population behavior.** The figure considers hypothetical architectures for a quorum sensing network with three QS systems. The first two, mimicking the architecture of *las* and *rhl,* are mutually reinforcing. The third system is both induced and repressed by the other two, matching the reported interactions of *pqs.* The panels show all three synthase expression levels as a function of population density. As the four panels show, even within the constraints of a particular architecture, a wide variety of responses are possible. (A) Baseline case with weak activation of system 3 by system 1 (low ğ›¼<sub>3,1</sub>). (B) Strong activation (high ğ›¼<sub>3,1</sub>). (C) Limited activation with weak repression of system 3 by system 2 (moderately negative ğ›¼<sub>3,2</sub>). (D) Damped activation (strongly negative ğ›¼<sub>3,2</sub>). (The data underlying this Figure and the code used to generate it can be found in https://doi.org/10.5281/zenodo.15808353.)

## References

1. GarcÃ­a-Reyes S, SoberÃ³n-ChÃ¡vez G, Cocotl-Yanez M. The third quorum-sensing system of *Pseudomonas aeruginosa:* Pseudomonas quinolone signal and the enigmatic PqsE protein. J Med Microbiol. 2020;69: 25â€“34. 

2. Rutherford S, Bassler B. Bacterial quorum sensing: Its role in virulence and possibilities for its control. Cold Spring Harb Perspect Med. 2012;2: a012427. 

3. Proctor C, McCarron P, Ternan N. Furanone quorum-sensing inhibitors with potential as novel therapeutics against *Pseudomonas aeruginosa.* J Med Microbiol. 2020;69: 195â€“206. 

4. Jakobsen T, Bjarnsholt T, Jensen P, Givskov M, HÃ¸iby N. Targeting quorum sensing in *Pseudomonas aeruginosa* biofilms: Current and emerging inhibitors. Future Microbiol. 2013;8: 901â€“921. 

5. Soukarieh F, Williams P, Stocks M, CÃ¡mara M. *Pseudomonas aeruginosa* quorum sensing systems as drug discovery targets: Current position and future perspectives. J Med Chem. 2018;61: 10385â€“10402. 

6. Tateda K. [*Pseudomonas aeruginosa* infection and the quorum-sensing mechanism]. Nihon Naika Gakkai Zasshi. 2005;94: 999â€“1004. 

7. Williams P, Winzer K, Chan W, CÃ¡mara M. Look whoâ€™s talking: Communication and quorum sensing in the bacterial world. Philos Trans R Soc Lond B Biol Sci. 2007;362: 1119â€“1134. 

8. Heurlier K, DÃ©nervaud V, Haas D. Impact of quorum sensing on fitness of *Pseudomonas aeruginosa.* Int J Med Microbiol. 2006;296: 93â€“102. 

9. Le Berre R, Faure K, Nguyen S, Pierre M, Ader F, Guery B. [Quorum sensing: A new clinical target for *Pseudomonas aeruginosa*?]. Med Mal Infect. 2006;36: 349â€“357. 

10. Juhas M, Eberl L, TÃ¼mmler B. Quorum sensing: The power of cooperation in the world of *Pseudomonas.* Environ Microbiol. 2005;7: 459â€“471. 

11. Donabedian H. Quorum sensing and its relevance to infectious diseases. J Infect. 2003;46: 207â€“214. 

12. Reuter K, Steinbach A, Helms V. Interfering with bacterial quorum sensing. Perspect Medicin Chem. 2016;8: 1â€“15. 

13. Yong Y, Zhong J. Impacts of quorum sensing on microbial metabolism and human health. Adv Biochem Eng Biotechnol. 2013;131: 25â€“61. 

14. Welsh M, Blackwell H. Chemical probes of quorum sensing: From compound development to biological discovery. FEMS Microbiol Rev. 2016;40: 774â€“794. 

15. De Sordi L, MÃ¼hlschlegel F. Quorum sensing and fungal-bacterial interactions in *Candida albicans:* A communicative network regulating microbial coexistence and virulence. FEMS Yeast Res. 2009;9: 990â€“999. 

16. Winzer K, Williams P. Quorum sensing and the regulation of virulence gene expression in pathogenic bacteria. Int J Med Microbiol. 2001;291: 131â€“143. 

17. Schuster M, Sexton D, Diggle S, Greenberg E. Acyl-homoserine lactone quorum sensing: From evolution to application. Annu Rev Microbiol. 2013;67: 43â€“63. 

18. Papaioannou E, Utari P, Quax W. Choosing an appropriate infection model to study quorum sensing inhibition in *Pseudomonas* infections. Int J Mol Sci. 2013;14: 19309â€“19340. 

19. Roy V, Adams B, Bentley W. Developing next generation antimicrobials by intercepting AI-2 mediated quorum sensing. Enzyme Microb Technol. 2011;49: 113â€“123. 

20. Cornforth D, Popat R, McNally L, Gurney J, Scott-Phillips T, Ivens A, et al. Combinatorial quorum sensing allows bacteria to resolve their social and physical environment. Proc Natl Acad Sci U S A. 2014;111: 4280â€“4284. 

21. Rattray J, Thomas S, Wang Y, Molotkova E, Gurney J, Varga J, et al. Bacterial quorum sensing allows graded and bimodal cellular responses to variations in population density. mBio. 2022;13: e0074522. 
