# The *las* and *rhl* Quorum Sensing Systems in *Pseudomonas aeruginosa* Form a Multi-Signal Reciprocal Network Which Can Tune Reactivity to Variations in Physical and Social Environments

## Supporting Information

### Literature Search

The PubMed database of the US National Institutes of Health was queried on 20 July 2021 using the query [`PubMed Search ("review"[Title/Abstract] OR "review"[Publication Type]) AND "quorum sensing"[Title] AND "pseudomonas aeruginosa"[Title/Abstract]`](https://pubmed.ncbi.nlm.nih.gov/?term=%28%22review%22%5BTitle%2FAbstract%5D+OR+%22review%22%5BPublication+Type%5D%29+AND+%22quorum+sensing%22%5BTitle%5D+AND+%22pseudomonas+aeruginosa%22%5BTitle%2FAbstract%5D&sort=), resulting in 76 results with publication dates from 1996 to 2021. Papers that incluced a daigram of the gene transcription networks for the *las* and *rhl* quorum sensing systems were further analyzed to show the interactions present on those diagrams. Tables S.[-@tbl:lasr] and S.[-@tbl:rhlr] show the results. Of the papers analyzed, all show the *las* system positively activating the *rhl* system, and none show the *rhl* system postively activating the *las* system.


| Paper | PMID | →_lasI_ | →_lasR_ | →_rhlI_ | →_rhlR_ | →elastase |
| ---- | :--: | :--: | :--: | :--: | :--: | :--: |
| [@GarcíaReyes2020] | [31794380](https://pubmed.ncbi.nlm.nih.gov/31794380/) | ● | ○ | ● | ● | ● |
| [@Rutherford2012] | [23125205](https://pubmed.ncbi.nlm.nih.gov/23125205/) | ● | ○ | ● | ● |  |
|[@Proctor2020] |[31971503](https://pubmed.ncbi.nlm.nih.gov/31971503/) | ● | ○ | ○ | ● |  |
|[@Jakobsen2013] |[23841636](https://pubmed.ncbi.nlm.nih.gov/23841636/) | ● | ○ | ● | ● |  |
|[@Soukarieh2018] |[29999316](https://pubmed.ncbi.nlm.nih.gov/29999316/) | ● | ● | ● | ● |  |
|[@Tateda2005] |[15926474](https://pubmed.ncbi.nlm.nih.gov/15926474/) | ● | ○ | ● | ○ | ● |
|[@Williams2007] |[19249239](https://pubmed.ncbi.nlm.nih.gov/19249239/) | ○ | ○ | ● | ● |  |
|[@Heurlier2006] |[16503417](https://pubmed.ncbi.nlm.nih.gov/16503417/) | ● | ○ | ● | ○ |  |
|[@LeBerre2006] |[16631332](https://pubmed.ncbi.nlm.nih.gov/16631332/) | ○ | ○ | ● | ● |  |
|[@Juhas2005] |[15816912](https://pubmed.ncbi.nlm.nih.gov/15816912/) | ● | ● | ● | ● | ● |
|[@Donabedian2003] |[12799145](https://pubmed.ncbi.nlm.nih.gov/12799145/) | ● | ○ | ● | ● | ● |
|[@Reuter2016] |[26819549](https://pubmed.ncbi.nlm.nih.gov/26819549/) | ● | ○ | ○ | ● | ● |
|[@Yong2013] |[22767136](https://pubmed.ncbi.nlm.nih.gov/22767136/) | ● | ○ | ● | ● | ● |
|[@Welsh2016] |[27268906](https://pubmed.ncbi.nlm.nih.gov/27268906/) |  |  | ● | ● | ● |
|[@DeSordi2009] |[19845041](https://pubmed.ncbi.nlm.nih.gov/19845041/) | ● | ○ | ● | ○ |  |
|[@Winzer2001] |[11437336](https://pubmed.ncbi.nlm.nih.gov/11437336/) | ● | ○ | ○ | ● | ● |
|[@Schuster2013] |[23682605](https://pubmed.ncbi.nlm.nih.gov/23682605/) | ● | ○ | ● | ● |  |
|[@Papaioannou2013] |[24065108](https://pubmed.ncbi.nlm.nih.gov/24065108/) | ● | ● | ○ | ● | ● |
|[@Roy2011] |[22112397](https://pubmed.ncbi.nlm.nih.gov/22112397/) | ● | ○ | ● | ● |  |

Table: lasr {#tbl:lasr}

<div custom-style="Caption"><p>
**Table S.[-@tbl:lasr].** Activation of QS genes by LasR/3‑oxo‑C<sub>12</sub>‑HSL in review of published literature. Solid dots indicate positive activation in the paper’s diagram of gene transcription, while hollow dots indicate that the diagram shows no effect. No diagrams indicated repression. Note that some papers made no attempt to indicate particular interactions; several, for example, concentrated strictly on the QS genes themselves and did not show the effect on downstream genes such as those for elastase.
</p></div>

| Paper | PMID | →_lasI_ | →_lasR_ | →_rhlI_ | →_rhlR_ | →elastase |
| ---- | :--: | :--: | :--: | :--: | :--: | :--: |
| [@GarcíaReyes2020] | [31794380](https://pubmed.ncbi.nlm.nih.gov/31794380/) | ○ | ○ | ○ | ○ | ● |
| [@Rutherford2012] | [23125205](https://pubmed.ncbi.nlm.nih.gov/23125205/) | ○ | ○ | ● | ○ |  |
|[@Proctor2020] |[31971503](https://pubmed.ncbi.nlm.nih.gov/31971503/) | ○ | ○ | ● | ○ | |
|[@Jakobsen2013] |[23841636](https://pubmed.ncbi.nlm.nih.gov/23841636/) | ○ | ○ | ● | ○ | |
|[@Soukarieh2018] |[29999316](https://pubmed.ncbi.nlm.nih.gov/29999316/) | ○ | ○ | ○ | ○ | |
|[@Tateda2005] |[15926474](https://pubmed.ncbi.nlm.nih.gov/15926474/) | ○ | ○ | ● | ○ | ● |
|[@Williams2007] |[19249239](https://pubmed.ncbi.nlm.nih.gov/19249239/) | ○ | ○ | ○ | ○ | |
|[@Heurlier2006] |[16503417](https://pubmed.ncbi.nlm.nih.gov/16503417/) | ○ | ○ | ● | ○ | |
|[@LeBerre2006] |[16631332](https://pubmed.ncbi.nlm.nih.gov/16631332/) | ○ | ○ | ○ | ○ | |
|[@Juhas2005] |[15816912](https://pubmed.ncbi.nlm.nih.gov/15816912/) | ○ | ○ | ● | ● | ● |
|[@Donabedian2003] |[12799145](https://pubmed.ncbi.nlm.nih.gov/12799145/) | ○ | ○ | ○ | ○ | ● |
|[@Reuter2016] |[26819549](https://pubmed.ncbi.nlm.nih.gov/26819549/) | ○ | ○ | ○ | ○ | ● |
|[@Yong2013] |[22767136](https://pubmed.ncbi.nlm.nih.gov/22767136/) | ○ | ○ | ● | ○ | ● |
|[@Welsh2016] |[27268906](https://pubmed.ncbi.nlm.nih.gov/27268906/) | ○ | ○ |  |  | ○ |
|[@DeSordi2009] |[19845041](https://pubmed.ncbi.nlm.nih.gov/19845041/) | ○ | ○ | ● | ○ | |
|[@Winzer2001] |[11437336](https://pubmed.ncbi.nlm.nih.gov/11437336/) | ○ | ○ | ● | ○ | ● |
|[@Schuster2013] |[23682605](https://pubmed.ncbi.nlm.nih.gov/23682605/) | ○ | ○ | ● | ○ |  |
|[@Papaioannou2013] |[24065108](https://pubmed.ncbi.nlm.nih.gov/24065108/) | ○ | ○ | ● | ○ | ● |
|[@Roy2011] |[22112397](https://pubmed.ncbi.nlm.nih.gov/22112397/) | ○ | ○ | ○ | ○ |  |

Table: rhlr {#tbl:rhlr}

<div custom-style="Caption"><p>
**Table S.[-@tbl:rhlr].** Activation of QS genes by RhlR/C<sub>4</sub>‑HSL in review of published literature. Same notation as previous table.
</p></div>

### Data Analysis

Gene expression data for *lasI,* *rhlI,* and *lasB* was collected every hour for a 24-hour period. Observations used for analysis were limited to a two-hour window that contained the peak expression level for each gene. Figures S.[-@fig:lasi_time], S.[-@fig:rhli_time], and S.[-@fig:lasb_time] show the full time course of expression levels and highlight the intervals used for analysis. Those windows were 8–10 hours, 3–5 hours, and 4–6 hours for *lasI,* *rhlI,* and *lasB,* respectively.

![lasi_time](Figures/lasi_time.svg){#fig:lasi_time}

<div custom-style="Caption"><p>
**Figure S.[-@fig:lasi_time]. Expression level of _lasI_ over time course of experiment.** Shaded regions highlight peak expression and indicate two-hour period used in analysis.
</p></div>

![rhli_time](Figures/rhli_time.svg){#fig:rhli_time}

<div custom-style="Caption"><p>
**Figure S.[-@fig:rhli_time]. Expression level of _rhlI_ over time course of experiment.** Shaded regions highlight peak expression and indicate two-hour period used in analysis.
</p></div>

![lasb_time](Figures/lasb_time.svg){#fig:lasb_time}

<div custom-style="Caption"><p>
**Figure S.[-@fig:lasb_time]. Expression level of _lasB_ over time course of experiment.** Shaded regions highlight peak expression and indicate two-hour period used in analysis.
</p></div>

### Multi-Signal Models

Figure 7 in the main text summarizes the predictions of the multi-signal models for *lasI* and *rhlI* expression. The following figures provide a more detailed comparison of the model predictions for all three genes.

![model_lasi](Figures/model_lasi.svg){#fig:model_lasi}

<div custom-style="Caption"><p>
**Figure S.[-@fig:model_lasi]. Multi-signal model for _lasI_ expression.** Panels compare model predictions to observations for all combinations of signal concentrations. Horizontal bars indicate model predictions, while plotted points show observed values.
</p></div>

![model_rhli](Figures/model_rhli.svg){#fig:model_rhli}

<div custom-style="Caption"><p>
**Figure S.[-@fig:model_rhli]. Multi-signal model for _rhlI_ expression.** Panels compare model predictions to observations for all combinations of signal concentrations. Horizontal bars indicate model predictions, while plotted points show observed values.
</p></div>

![model_lasb](Figures/model_lasb.svg){#fig:model_lasb}

<div custom-style="Caption"><p>
**Figure S.[-@fig:model_lasb]. Multi-signal model for _lasB_ expression.** Panels compare model predictions to observations for all combinations of signal concentrations. Horizontal bars indicate model predictions, while plotted points show observed values.
</p></div>
### Signal Dynamics

We analyze signal dynamics using the model from the main text where the per-capita single production rater is assumed to be proportional to the synthase expression level, $E_i(\mathbf{S})$. The proportionality constant is $c_i$.

$$
\frac{\mathrm{d}S_i}{\mathrm{dt}} \ \ = \ \ c_i E_i(\mathbf{S})\cdot N \ \ - \ \  \delta_i \cdot S_i \ \ - \ \  m \cdot S_i
$$

We consider the equilibrium signal concentration (where $\mathrm{d}S/\mathrm{dt} = 0$) and normalize to the decay rate of  C<sub>4</sub>‑HSL ($\delta_2$). When there is no mass transfer ($m = 0$), these simplifications result in an equation for  C<sub>4</sub>‑HSL,

$$
0 \ \ = \ \ \frac{ c_2 }{ \delta_2 } E_2(\mathbf{S})\cdot N \ \ - \ \ S_2
$$

which can be solved for $c_2 / \delta_2$ in terms of *rhlI* expression $E_2(\mathbf{S})$, density $N$, and C<sub>4</sub>‑HSL concentration $S_2$. The corresponding equation for 3‑oxo‑C<sub>12</sub>‑HSL includes an additional factor $\delta_1/\delta_2$ which, from [@Cornforth2014], we take to be approximately 1.7.

$$
0 \ \ = \ \ \frac{ c_1 }{ \delta_2 } E_1(\mathbf{S})\cdot N \ \ - \ \ \frac{ \delta_1 }{ \delta_2 } S_1
$$

Data from [@Rattray2022] includes measurements of equilibrium signal concentrations at multiple population densities. We combine those measurements of $N$ and $S_i$ with our model’s estimate of synthase expression level $E_i(\mathbf{S})$ and use non-linear least squares to estimate the proportionality constants.

| Signal _i_                   | Proportionality Constant $c_i/\delta_2$ |
| :--------------------------- | --------------------------------------: |
| **3‑oxo‑C<sub>12</sub>‑HSL** |                              1.3 nM/RLU |
| **C<sub>4</sub>‑HSL**        |                              2.5 nM/RLU |

Table: const {#tbl:const}

<div custom-style="Caption"><p>
**Table S.[-@tbl:const].** Estimated proportionality constants that relate synthase expression levels to per-capita signal production rates. Final column shows adjusted R<sup>2</sup> of non-linear least squares estimate.
</p></div>

![constants](Figures/constants.svg){#fig:constants}

<div custom-style="Caption"><p>
**Figure S.[-@fig:constants]. Equilibrium signal concentration predicted using proportionality constants.** Individual data points show experimental observations and dashed lines indicate model predictions.
</p></div>
### Analytic Solutions for Equilibrium

It is possible to derive analytic solutions of Equation 2 (main text) for equilibrium concentrations in all architectures; however, the results are not especially helpful for deriving insights into the system behavior. For example, the independent architecture, which is the simplest considered, has the following equilibrium concentration of 3‑oxo‑C <sub>12</sub>‑HSL.
$$
S_1^* = \frac{
\begin{align}
c_1\,N\,(\alpha_{1,0} +\alpha_{1,1}) \\ - K_{1,1}\,(m+\delta_1)
\end{align}
\;+\sqrt{
\begin{aligned}
{K_{1,1} }^2 \,{\delta_1 }^2 +2\,{K_{1,1} }^2 \,\delta_1 \,m+{K_{1,1} }^2 \,m^2
+2\,K_{1,1} \,N\,\alpha_{1,0} \,c_1 \,\delta_1 \\
+2\,K_{1,1} \,N\,\alpha_{1,0} \,c_1 \,m -2\,K_{1,1} \,N\,\alpha_{1,1} \,c_1 \,\delta_1 -2\,K_{1,1} \,N\,\alpha_{1,1} \,c_1 \,m \\
+N^2 \,{\alpha_{1,0} }^2 \,{c_1 }^2 +2\,N^2 \,\alpha_{1,0} \,\alpha_{1,1} \,{c_1 }^2
+N^2 \,{\alpha_{1,1} }^2 \,{c_1 }^2
\end{aligned}
}
}{2\,{\left(\delta_1 +m\right)}}
$$


### Normalizing Alternate QS Architectures

The main text analyzes hypothetical, alternative architectures by eliminating the influence of specific signals on specific genes. For example, the hierarchical architecture nullifies the influence of C<sub>4</sub>‑HSL on *lasI* without modifying the effect of 3‑oxo‑C<sub>12</sub>‑HSL on *lasI.* This change necessarily reduces the maximum expression level of *lasI,* and that reduction partially explains the different *lasB* response in a hierarchical architecture. Reducing maximum *lasI* expression alone, however, does not explain all of the differences in the *lasB* response. To expose those additional differences, we make additional adjustments to the model. In particular, we increase the expression of *lasI* due to 3‑oxo‑C<sub>12</sub>‑HSL to precisely compensate for the loss of expression due to C<sub>4</sub>‑HSL. Table S.[-@tbl:architectures] shows the full set of adjustments required to normalize the maximum synthase expression levels across all architectures.


| Gene | Signal | Parameter | Derivation | Reciprocal Architecture | Hierarchical  Architecture | Independent Architecture |
|-|-|--------|:------:|:------:|:------:|:------:|
| *lasI* | 3‑oxo‑C<sub>12</sub>‑HSL | Max fold-change | (*ɑ*<sub>1,1</sub> + *ɑ*<sub>1,0</sub>) / *ɑ*<sub>1,0</sub> | 38 × | 73 × | 73 × |
|  | C<sub>4</sub>‑HSL | Max fold-change | (*ɑ*<sub>1,2</sub> + *ɑ*<sub>1,0</sub>) / *ɑ*<sub>1,0</sub> | 6.4 × |            1 ×             | 1 × |
|  | Combined | Max fold-change | (*ɑ*<sub>1,1,2</sub> + *ɑ*<sub>1,0</sub>) / *ɑ*<sub>1,0</sub> | 30 × | 1 × | 1 × |
| *rhlI* | 3‑oxo‑C<sub>12</sub>‑HSL | Max fold-change | (*ɑ*<sub>2,1</sub> + *ɑ*<sub>2,0</sub>) / *ɑ*<sub>2,0</sub> | 35 × | 35 × | 1 × |
| | C<sub>4</sub>‑HSL | Max fold-change | (*ɑ*<sub>2,2</sub> + *ɑ*<sub>2,0</sub>) / *ɑ*<sub>2,0</sub> | 6.4 × | 6.4 × | 66 × |
| | Combined | Max fold-change | (*ɑ*<sub>2,1,2</sub> + *ɑ*<sub>1,0</sub>) / *ɑ*<sub>1,0</sub> | 27 × | 27 × | 1 × |

Table: architectures {#tbl:architectures}

<div custom-style="Caption"><p>
**Table S.[-@tbl:architectures]. Models of hierarchical and independent architectures can be normalized to ensure that maximum synthase expression is the same for all architectures.** Parameters are the same as those in the main text but with increased values where appropriate.
</p></div>

The result of these adjustments, shown in Figure [-@fig:lasb_scaled_heatmaps] shows the same qualitative differences as the unnormalized models in the main text. The reciprocal architecture remains the most sensitive to changes in population density and the most robust to signal loss from mass transfer.

![lasb_scaled_heatmaps](Figures/lasb_scaled_heatmaps.svg){#fig:lasb_scaled_heatmaps}

<div custom-style="Caption"><p>
**Figure S.[-@fig:lasb_scaled_heatmaps]. The reciprocal QS architecture is more sensitive to population density and more robust to environmental interference.** Plots are the same as those in the corresponding figure in the main text, but show the normalized models.
</p></div>



## References



